### ISSUE TITLE
Panic while typing `du -ma`

### ISSUE THREAD

**Issue by Cyborus04** [nushell/nushell#4024]
### Describe the bug

If you type `du -ma` it panics with 
```
thread 'main' panicked at 'index out of bounds: the len is 2 but the index is 2', crates\nu-parser\src\parse.rs:1575:68
stack backtrace:
   0:     0x7ff63c65999f - git_repository_get_namespace
   1:     0x7ff63c6792ca - git_repository_get_namespace
   2:     0x7ff63c653c78 - git_repository_get_namespace
   3:     0x7ff63c65caa6 - git_repository_get_namespace
   4:     0x7ff63c65c589 - git_repository_get_namespace
   5:     0x7ff63c65d105 - git_repository_get_namespace
   6:     0x7ff63c65ccf1 - git_repository_get_namespace
   7:     0x7ff63c65a2ef - git_repository_get_namespace
   8:     0x7ff63c65cc49 - git_repository_get_namespace
   9:     0x7ff63c803fc0 - git_proxy_options_init
  10:     0x7ff63c803f87 - git_proxy_options_init
  11:     0x7ff63bcdfcd5 - git_odb_object_data
  12:     0x7ff63bceca59 - git_odb_object_data
  13:     0x7ff63bcf0f5f - git_odb_object_data
  14:     0x7ff63bcf3fe3 - git_odb_object_data
  15:     0x7ff63bcf6ac1 - git_odb_object_data
  16:     0x7ff63b891b30 - <unknown>
  17:     0x7ff63b88c8cc - <unknown>
  18:     0x7ff63b8cebe6 - git_patch_owner
  19:     0x7ff63b8b9c13 - <unknown>
  20:     0x7ff63b8b7bfa - <unknown>
  21:     0x7ff63b8ba6a3 - <unknown>
  22:     0x7ff63b8b39cc - <unknown>
  23:     0x7ff63b8b2147 - <unknown>
  24:     0x7ff63b8b492c - <unknown>
  25:     0x7ff63b8b4b42 - <unknown>
  26:     0x7ff63b8992e9 - <unknown>
  27:     0x7ff63b884e39 - <unknown>
  28:     0x7ff63b883926 - <unknown>
  29:     0x7ff63b881662 - <unknown>
  30:     0x7ff63b88100a - <unknown>
  31:     0x7ff63b881057 - <unknown>
  32:     0x7ff63c65d65a - git_repository_get_namespace
  33:     0x7ff63b881039 - <unknown>
  34:     0x7ff63c7e5200 - git_proxy_options_init
  35:     0x7fff136f7034 - BaseThreadInitThunk
  36:     0x7fff13842651 - RtlUserThreadStart
```

### How to reproduce

1. Open Nushell
2. Type `du -ma` (don't even need to press Enter)

### Expected behavior

Not crashing, allowing `du -ma` to exist in the prompt

### Screenshots

![nushell error](https://user-images.githubusercontent.com/87248184/133802144-9a50ce1c-8fbe-4c65-afc8-c87b42569ab7.png)

### Configuration

|key|value|
|-|-|
|version|0.36.0|
|short_commit|991a4801|
|commit_hash|991a4801b1f6f451876916bb6453749a460d3b17|
|commit_date|2021-08-24 18:01:17 +00:00|
|build_os|windows-x86_64|
|rust_version|rustc 1.54.0 (a178d0322 2021-07-26)|
|rust_channel|stable-x86_64-pc-windows-msvc (default)|
|cargo_version|cargo 1.54.0 (5ae8d74b3 2021-06-22)|
|pkg_version|0.36.0|
|build_time|2021-08-24 18:32:49 +00:00|
|build_rust_channel|release|
|features|clipboard-cli, ctrlc, dataframe, default, rustyline, term, trash, uuid, which, zip|
|installed_plugins|binaryview, chart bar, chart line, fetch, from bson, from sqlite, inc, match, post, ps, query json, s3, selector, start, sys, textview, to bson, to sqlite, tree, xpath|

### Additional context

Using Starship prompt

**Comment by kubouch** [comment]
Seems similar to #4016

**Comment by ahkrr** [comment]
I think this Issue can be closed because https://github.com/nushell/nushell/pull/4107 should have fixed it.

**Comment by kubouch** [comment]
Indeed, does not happen anymore for me. I'll close it, please reopen if it becomes an issue again.

@ahkrr You could have put "Closes #4024, #4016" line to your PR descriptions and these would get closed automatically. Thanks for fixing it!
